community/modules/scheduler/schedmd-slurm-gcp-v5-login/variables.tf

```diff
@@ -117,18 +117,6 @@ variable "access_config" {
   default = []
 }
 
-variable "nic_type" {
```

Comment:
I don't understand why remove the nic_type and the bandwidth tier, along with access_config, these used to work, right?

Reply:
No: https://github.com/GoogleCloudPlatform/hpc-toolkit/pull/1798#discussion_r1341799392

Reply:
got it, this was erroneously introduced in the 5.9.0 branch.